mqtt/mqtt-edgehub/src/auth/authorization/edgehub.rs

```diff
@@ -252,14 +252,15 @@ where
     }
 
     fn update(&mut self, update: Box<dyn Any>) -> Result<(), Self::Error> {
-        match update.downcast::<Vec<ServiceIdentity>>() {
+        match update.downcast::<AuthorizerUpdate>() {
             Ok(service_identities) => {
                 debug!(
                     "service identities update received. Service identities: {:?}",
                     service_identities
                 );
 
                 self.service_identities_cache = service_identities
+                    .0
```

Comment:
this turns it from AuthorizerUpdate -> Vec<IdentityUpdate>?

Reply:
right, AuthorizerUpdate is a tuple struct, so you can access its fields like it is a tuple.